packages/picasso/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -100,7 +100,7 @@ export const RichTextEditor = forwardRef<HTMLDivElement, Props>(
     const toolbarRef = useRef<HTMLDivElement>(null)
     const editorRef = useRef<HTMLDivElement>(null)
     const { dispatch, state } = useTextEditorState()
-    const [isEditorFocused, setIsEditorFocused] = useState(autofocus!) // eslint-disable-line @typescript-eslint/no-non-null-assertion
+    const [isEditorFocused, setIsEditorFocused] = useState(autoFocus!) // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
Is it more meaningful to provide a boolean result rather than non-null force(and disable rules)? WDYT?
```suggestion
    const [isEditorFocused, setIsEditorFocused] = useState(!!autoFocus)
```

Reply:
@ozgurkececioglu we have `autoFocus` in default values, so in this case, we know for sure, that it is always a boolean value.  We use it like this in the rest of the codebase, but I agree that it is not that nice and I would prefer to have the default value in destructed prop, but I think it is not in the scope of this ticket and we should change it across whole codebase in different task